test/helpers/otlp/attributes.go

```diff
@@ -0,0 +1,51 @@
+package otlp
+
+const (
+	NodeName           = "node.name"
+	ClusterID          = "cluster.id"
+	K8sNamespaceName   = "k8s.namespace.name"
+	K8sPodName         = "k8s.pod.name"
+	K8sContainerName   = "k8s.container.name"
+	OpenshiftLogSource = "openshift.log.source"
+	OpenshiftLogType   = "openshift.log.type"
+)
+
+func (r Resource) NodeNameAttribute() string {
```

Comment:
I think this functions would be more valuable to call FindStringValue then needing to pair them together inthe test.  As-is its a function to return a constant that is already public

Reply:
This is meant to grow and end up as a documented list of all attributes we are using and whether they are logRecord or resource attributes.   Id prefer to keep as is for now.   Makes for easier testing and logical reference, imho.

Reply:
Maybe a need to refactor after merge.... but

* They can still be constants that we use for documenting, they just would have to live in a different package
* the usage as-is IMO is a little verbose  for example:
```
r.FindStringValue(ClusterID)
```
instead of just:
```
r.ClusterID()
```
One has to lookup what the attribute they want instead of simply inspecting the available things.  We know there is a fixed set of items we may wish to retrieve and the design still allows to get those which are accessed less.